### ISSUE TITLE
Problem pom not found version 1.0.2

### ISSUE THREAD

**Issue by FarshidRoohi** [florent37/MultiplatformPreferences#8]
```
Could not find com.github.florent37:multiplatform-preferences-android:1.0.2.
Searched in the following locations:
  - https://dl.bintray.com/florent37/maven/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://dl.bintray.com/florent37/maven/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
  - https://dl.bintray.com/suparnatural/kotlin-multiplatform/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://dl.bintray.com/suparnatural/kotlin-multiplatform/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
  - https://maven.google.com/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://maven.google.com/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
  - https://kotlin.bintray.com/kotlinx/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://kotlin.bintray.com/kotlinx/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
  - https://dl.google.com/dl/android/maven2/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://dl.google.com/dl/android/maven2/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
  - https://jcenter.bintray.com/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://jcenter.bintray.com/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
  - https://repo.maven.apache.org/maven2/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.pom
  - https://repo.maven.apache.org/maven2/com/github/florent37/multiplatform-preferences-android/1.0.2/multiplatform-preferences-android-1.0.2.jar
```

**Comment by florent37** [comment]
fixed in 1.2.0, can you try it ?

**Comment by FarshidRoohi** [comment]
@florent37 
Updated to 1.2.0
import not found
`import com.github.florent37.preferences.Preferences
`

when run project and see this error : 

```
Unable to find a matching variant of com.github.florent37:multiplatform-preferences-android:1.2.0:
  - Variant 'android-releaseApiElements' capability com.github.florent37:multiplatform-preferences-android:1.2.0:
      - Incompatible attributes:
          - Required com.android.build.api.attributes.BuildTypeAttr 'debug' and found incompatible value 'release'.
          - Required org.gradle.usage 'java-runtime' and found incompatible value 'java-api'.
      - Other attributes:
          - Found com.android.build.api.attributes.VariantAttr 'release' but wasn't required.
          - Required com.android.build.gradle.internal.dependency.AndroidTypeAttr 'Aar' and found compatible value 'Aar'.
          - Found org.gradle.status 'release' but wasn't required.
          - Required org.jetbrains.kotlin.platform.type 'androidJvm' and found compatible value 'androidJvm'.
  - Variant 'android-releaseRuntimeElements' capability com.github.florent37:multiplatform-preferences-android:1.2.0:
      - Incompatible attribute:
          - Required com.android.build.api.attributes.BuildTypeAttr 'debug' and found incompatible value 'release'.
      - Other attributes:
          - Found com.android.build.api.attributes.VariantAttr 'release' but wasn't required.
          - Required com.android.build.gradle.internal.dependency.AndroidTypeAttr 'Aar' and found compatible value 'Aar'.
          - Found org.gradle.status 'release' but wasn't required.
          - Required org.gradle.usage 'java-runtime' and found compatible value 'java-runtime'.
          - Required org.jetbrains.kotlin.platform.type 'androidJvm' and found compatible value 'androidJvm'.
  - Variant 'metadata-api' capability com.github.florent37:multiplatform-preferences-android:1.2.0:
      - Incompatible attributes:
          - Required org.gradle.usage 'java-runtime' and found incompatible value 'kotlin-api'.
          - Required org.jetbrains.kotlin.platform.type 'androidJvm' and found incompatible value 'common'.
      - Other attributes:
          - Required com.android.build.api.attributes.BuildTypeAttr 'debug' but no value provided.
          - Required com.android.build.gradle.internal.dependency.AndroidTypeAttr 'Aar' but no value 
provided.
          - Found org.gradle.status 'release' but wasn't required.

```
